src/components/VoteRow.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+const Wrapper = styled.div`
+  width: 150px;
+  margin: 10px;
+`;
+const Button = styled.button`
+  width: 100px;
+`;
+
+export default function VoteRow({ candidate, order, setCandidates }) {
+  const { id } = candidate;
```

Comment:
`name`과 `voteCount`도 뽑아와주세요 ~

```suggestion
  const { id,name,voteCount } = candidate;
```